src/hera/workflows/script.py

```diff
@@ -62,6 +62,9 @@
     from typing_extensions import Annotated  # type: ignore
 
 
+_DEFAULT_ARTIFACT_INPUT_DIRECTORY = "/tmp/hera/inputs/artifacts/"
```

Comment:
Is this folder created by Hera when annotations are used? I think writing to it might fail if it does not exist